content/blog/elm-review-v2/index.md

```diff
@@ -0,0 +1,269 @@
+---
+title: elm-review v2!
+date: '2020-04-30T00:00:00.000Z'
+---
+
+Today I am very excited to release `elm-review` 2.0.0 and to share its new features!
+
+tl;dr: Here is the list of the introduced features:
+
+#### New review capabilities!
+
+- Project rules!
```

Comment:
```suggestion
- Project-wide rules!
```
maybe? Reading "Project rules", my first thought was that you could visit non-code resources at least 🙂

Reply:
That's what I called it at first :D

> my first thought was that you could visit non-code resources at least :slightly_smiling_face:

You can :D. You can visit:
- the elm.json file
- the README
- access the dependencies' information

In the case of project rules, you can even report errors for the two first ones.

So I think this still makes sense. Especially since I now call it like this everywhere in the documentation, in contrast to module rules which can access the same information but only report errors inside a module (which is a code-resource).